app/Http/Controllers/BeatmapsController.php

```diff
@@ -25,7 +24,7 @@
 class BeatmapsController extends Controller
 {
     const DEFAULT_API_INCLUDES = ['beatmapset.ratings', 'failtimes', 'max_combo'];
-    const DEFAULT_SCORE_INCLUDES = ['user', 'user.country', 'user.cover'];
+    const DEFAULT_SCORE_INCLUDES = ['user', 'user.country', 'user.cover', 'beatmap.owners'];
```

Comment:
I don't think including same beatmap multiple times when fetching leaderboard would be useful?

the commit log says for userScore but then userScore also included this manually...?